VideoLocker/src/main/java/org/edx/mobile/view/adapters/MyCourseAdapter.java

```diff
@@ -203,15 +204,14 @@ public BaseViewHolder getTag(View convertView) {
     }
 
     private static class ViewHolder extends BaseViewHolder {
-        NetworkImageView courseImage;
+        ImageView courseImage;
         TextView courseTitle;
         TextView schoolCodeTv;
         TextView orgCodeTv;
         TextView orSymbolTv;
         TextView starting_from;
         LinearLayout new_course_content;
-        LinearLayout starting_from_layout;
-        RelativeLayout certificateBanner;
+        View certificateBanner;
```

Comment:
If we are making the type for this more generic, then the `new_course_content` type should also be changed to be `View`. Similarly, the `facePileContainer` type can be changed from `LinearLayout` (which is implementation-specific) to `ViewGroup` (as we have defined it as a container).